terraform/.rubocop.yml

```diff
@@ -1 +1,7 @@
 inherit_from: ../.rubocop.yml
+
+Layout/FirstArrayElementIndentation:
```

Comment:
Question: should we just do this in `../.rubocop.yml`? `wc -l */.rubocop.yml` suggests modules tend not to customize.
If doing so would make a big diff of lint fixes, maybe handle as a separate cleanup PR?

Reply:
Updated here https://github.com/dependabot/dependabot-core/pull/3649